index.html

```diff
@@ -147,13 +147,24 @@ <h4>Chrome</h4>
 						<li>Turn on Developer mode in the top right corner.</li>
 						<li>Drag and drop the downloaded ZIP file into the page.</li>
 						</ul>
-						<h4>Firefox</h4>
+					</div>
+					<div class="col-base col-lg col-lg-4">
+						<h4>Firefox (standard)</h4>

```

Comment:
Maybe this should be called "Firefox (signed)"?

Reply:
Unless I also changed the name in the table to (signed), I'm not sure end-user would know what signed meant, especially if one day the signing issue fixes itself and there are no unsigned builds in the table.